hal/ia32/cpu.c

```diff
@@ -46,7 +46,7 @@ void hal_cpuJump(addr_t addr)
 	syspage->kernel = *(unsigned int *)((char *)syspage + sizeof(syspage_hal_t)); /* Kernel */
 	syspage->kernelsz = 0xc0000;                                                  /* Kernel size */
 	syspage->console = 0;                                                         /* Console */
-	syspage->arg[0] = '\0';                                                       /* Args */
+	hal_strcpy(syspage->arg, "Xpc-tty Xpc-ata Xpsh;-i;/etc/rc.psh");              /* Args */
```

Comment:
we need `uart16550` for CI to work, so that's not a correct solution :/

Reply:
You're right. I forgot about the CI using `uart16550`. In that case I really think that we need `kernelcmd/kernelargs` command in `plo` to control arguments passed to the kernel. We could have separate `plo` scripts for the CI and the default image. The CI script would have different kernel args (with `uart16550` instead of `pc-tty` + we could remote the `wait 2000` command.